src/components/InvoiceTimer/InvoiceTimer.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import * as React from 'react';
+import styled from 'styled-components';
+
+const TimeText = styled.p`
```

Comment:
Very similar to the `Small` atom which already exists.  
perhaps import that, and style that component further instead, or just re-use that component directly.